src/task/SubscribeAndReceiveSubscribeAckTask.ts

```diff
@@ -88,14 +109,36 @@ export default class SubscribeAndReceiveSubscribeAckTask extends BaseTask {
       isSendingStreams,
       this.context.videoStreamIndex.localStreamDescriptions(),
       // TODO: handle check-in mode, or remove this param
-      true
+      true,
+      compressedSDPOffer
     );
     this.context.logger.info(`sending subscribe: ${JSON.stringify(subscribe)}`);
     this.context.signalingClient.subscribe(subscribe);
 
     const subscribeAckFrame = await this.receiveSubscribeAck();
     this.context.logger.info(`got subscribe ack: ${JSON.stringify(subscribeAckFrame)}`);
+
+    let decompressedText = '';
+    if (subscribeAckFrame.compressedSdpAnswer && subscribeAckFrame.compressedSdpAnswer.length > 0) {
+      decompressedText = this.textCompressor.decompress(
+        subscribeAckFrame.compressedSdpAnswer,
+        this.context.previousSdpAnswerAsString
+      );
+
+      if (decompressedText.length === 0) {
+        this.context.sdpAnswer = '';
+        this.context.previousSdpAnswerAsString = '';
+        this.logAndThrow(`Error occurred while trying to decompress the SDP answer.`);
+      }
+
+      this.context.logger.info(
```

Comment:
Should this be info or debug level :D

Reply:
I kept it as info as it helped to identify if there were any issues with compression or not. Also it acts as a hint to inform us if the compression was enabled for the Account or not.